explain/insights/io-heavy.md

```diff
@@ -0,0 +1,17 @@
+---
+explain_insight: 'i/o-heavy'
```

Comment:
Is the “/“ in the insight name intentional? (i.e. isn’t it “io-heavy” without the slash)

Reply:
The annotation "slug" in pganalyze is "i/o-heavy" right now. I felt that read better than the alternatives. That's what this is referring to, no?